docs/rootstock/setup_regtest.md

```diff
@@ -0,0 +1,154 @@
+  </svg>
+</div>
+  <a href="https://optimism.io"><img alt="Optimism" src="https://raw.githubusercontent.com/ethereum-optimism/brand-kit/main/assets/svg/OPTIMISM-R.svg" width=600></a>
+  <br />
+  <h3><a href="https://optimism.io">Optimism</a> is <del>Ethereum</del> <ins>Rootstock</ins>, scaled.</h3>
+  <br />
+</div>
+
+# Guide to setup local L1 and L2 development nodes
+
+You will need **5 TTYs** for running the both **L1** and **L2** nodes.
```

Comment:
```suggestion
You will need **6 TTYs** for running the both **L1** and **L2** nodes.
```

Reply:
5 to run the nodes.
6th is for testing ;)